### DISCUSSION TITLE
Need guide to add custom domain

### DISCUSSION THREAD

**Discussion by alamzaib-farooq** [datalust/seq-tickets#1613] [Help]
I have an Azure virtual machine and I installed seq on it. It was working fine on http://localhost:5341 and giving me logging information. Then I changed the listen URL to my domain. I confirmed URL using **seq config get -k api.listenUris** and it's showing me current listening url seq.mydomain.com. But I am not able to open the dashboard. I have attached a screenshot of my DNS zone info. I did not add any port in Azure inbound rule. Please guide me how to fix this. 
![Screenshot 2022-07-17 160533](https://user-images.githubusercontent.com/25901497/179395303-f2906aa3-70e3-4b6d-a6a2-b6bfc0df60c8.png)

**Comment by nblumhardt** [comment]
Hi @alamzaib-farooq; there's some additional info in:

https://docs.datalust.co/docs/azure-installation

which may help?

Checking that the port you're exposing is open in the Windows Firewall is also important.

Best regards,
Nick
